.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!-- Please remove the appropriate section.
+For example, if this is a new feature, remove the "Bug Report" section -->
+
+# Bug Report
+
+Fixes issue #<NUMBER>
+
+## Summary
+<!-- Describe your fix, including an explanation of how to reproduce the bug
+before and after the PR.-->
+
+## Checklist
+- [ ] Signed all commits for DCO
+- [ ] Added tests
+- [ ] Updated documentation (as needed)
+- [ ] Updated migration guide (as needed)
+- [ ] `codecheck` passed (See
```

Comment:
sdformat doesn't have `make codecheck`. Can you add "run `sh tools/code_check.sh` from the source directory" here?

Reply:
How's this? 04f46cc323e11a74271ece8dac1fc493280e4595

Reply:
Looks good, thanks!